bridges/TwitterBridge.php

```diff
@@ -481,7 +482,7 @@ private static function compareTweetId($tweet1, $tweet2) {
 
 	//The aim of this function is to get an API key and a guest token
 	//This function takes 2 requests, and therefore is cached
-	private function getApiKey() {
+	private function getApiKey($dumpToken) {
```

Comment:
What is the purpose of dumpToken? Flag to ignore cache with api key and guest token?

Reply:
It is to invalidate the cached guest token.

I could also delete the token before calling getApiKey() the second time in getApiContents(), and not pass a flag.